src/index.html

```diff
@@ -2,13 +2,106 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta 
+      http-equiv="X-UA-Compatible" 
+      content="ie=edge"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="stylesheet" 
+      href="./style.css"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a 
+        href="http://localhost:8080"
+        class="header-icon">
+        <img
+        src="./images/logo.png" 
+        alt="header-icon"
+      >
+      </a>
+      
+      <nav class="nav">
+        <ul class ="nav-list">
+          
+          <li class="nav-item"> 
+            <a
+              href="#Apple"
+              class="nav-link is-active"
+            >Apple</a> 
+          </li>
```

Comment:
```suggestion
          <li class="nav-item"> 
            <a
              href="#Apple"
              class="nav-link is-active"
            >
              Apple
            </a> 
          </li>
```
check your code style, fix all such cases